app/inflation/README.md

```diff
@@ -0,0 +1,97 @@
+
+# UMEE Tokenomic Restructuring Proposal
+
+## Introduction
+
+The UMEE Tokenomic Restructuring proposal aims to optimize the tokenomics of the $UMEE token, which is issued following the standard Cosmos SDK mechanism. The goal is to progressively limit the token issuance in an algorithmic way, taking into account various factors such as network security, adoption of liquid staking, and additional income sources for stakers through interchain security and chain-based revenue streams.
+
+## Design Overview
+
+1. The $UMEE 2.0 Tokenomics will adopt a similar Cosmos dynamic inflation mechanism based on the bonding rate of the network. This allows the Umee chain to naturally adjust the inflation rate to drive staking activities and reach the targeted bonding rate.
+
+2. To initiate the new tokenomics, the maximum and minimum inflation rates will decrease by 25%. Subsequent decreases will occur every two years.
+
+   - Current min and max inflation rates: 7% - 14%
+   - New max and min inflation rates at the start: 5.6% - 11.2%
+
+3. The max and min inflation rates will decrease by 25% every two years in inflation cycles. This is in anticipation of the growth of the Umee ecosystem and wider adoption of liquid staking.
+
+   - Each inflation cycle will have a variable yearly total amount of newly emitted tokens.
+   - At the end of each inflation cycle, the `min_inflation_rate`, `max_inflation_rate` will be decreased by 25%.
+   - The inflation rate change speed will be accelerated from 1 year to 6 months.
```

Comment:
Maybe reference the existing `x/staking` module here, if it contains the parameter being modified

Reply:
We are modifying the mint module params 

Okay I will add link of mint module params  here